client/src/components/Banner.jsx

```diff
@@ -48,17 +51,19 @@ const Banner = () => (
           </form>
         </div>
       </HeaderBox>
-      <HeaderBox>
-        <div>
-          <img alt="Target" src={require('../svg/user-circle.svg').default} />
-          <span>
-            <Arrow alt="Arrow" src={require('../svg/arrow.svg').default} />
-          </span>
-        </div>
-        <div>
-          <span>Sign In</span>
-        </div>
-      </HeaderBox>
+      {/* <HeaderBox>
```

Comment:
Consider getting rid of your commented out code, unless you are keeping it for reference.